server/sonar-db-migration/src/test/java/org/sonar/server/platform/db/migration/version/v63/PopulateDefaultPermTemplateColumnsOfOrganizationsTest.java

```diff
@@ -120,6 +134,22 @@ public void execute_sets_project_perm_template_from_global_property_and_view_per
     verifyPropertiesDoNotExist();
   }
 
+  @Test
+  public void execute_should_update_kee_when_old_kee_is_too_long() throws SQLException {
```

Comment:
it would be more accurate to test that key is updated when length is 41, when it's more but not when it's 40.

In the current state of the test, you may update the kee when it's 33 (which would be unexpected) or not update it when it's 41 (not expected either) or update it only when it's 100.

According to existing tests, we can only know for sure that renaming is not done when key's length is 3 :)